src/App.tsx

```diff
@@ -1,18 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination/Pagination';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42)
   .map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [page, setPage] = useState(1);
+  const [itemsPerPage, setItemsPerPage] = useState(5);
+
+  const onSetPage = (currentPage: number) => {
+    setPage(currentPage);
+  };
+
+  const startItemPerPage = (page - 1) * itemsPerPage;
+  const visibleItems = [...items].splice(startItemPerPage, itemsPerPage);
+  const from = startItemPerPage + 1;
+  const to = startItemPerPage + itemsPerPage;
+
   return (
     <div className="container">
       <h1>Items with Pagination</h1>
 
       <p className="lead" data-cy="info">
-        Page 1 (items 1 - 5 of 42)
+        {`Page ${page} (items ${from} - ${to > 42 ? 42 : to} of 42)`}
```

Comment:
Nope. What if you will have pages !== 42? For example 95. Need to think a bit of a proper logic here.